doc/lsst.analysis.tools/action-types.rst

```diff
@@ -0,0 +1,32 @@
+============
+Action Types
+============
+
+Here is a list of the current actions implemented in ``analysis tools``, please look at these carefully before
+adding new actions to avoid duplication.
+
+Plot actions are covered in a separate :doc:`section <plot-types>`.
+
+Vector Actions
+==============
+
+.. automodapi:: lsst.analysis.tools.actions.vector
+    :no-inherited-members:
+    :skip: ConfigurableActionField
+    :skip: ConfigurableActionStructField
+    :skip: DictField
+    :skip: Field
+    :skip: Vector
+    :skip: VectorAction
```

Comment:
I think you can delete these as they're now generating a build warning indicating that these objects aren't available in the namespace.

My guess is that these objects are now dropped through a properly specified `__all__` in the sub-modules that get import into `lsst.analysis.tools.actions.vector`.